sites/public/src/pages/applications/household/add-members.tsx

```diff
@@ -27,14 +27,14 @@ const ApplicationAddMembers = () => {
   const { conductor, application, listing } = useFormConductor("addMembers")
   const router = useRouter()
   const currentPageSection = 2
-  const householdSize = application.householdMembers.length + 1
+  const householdSize = parseInt(application.householdMembers.length) + 1
```

Comment:
Question: What is the benefit of this?

Reply:
It was a Typescript error, now that the application is the JSON-ified and re-parsed object Typescript wasn't sure about the type here to know if it was chill to add a number

Reply:
Ah makes sense, thanks!